### PR TITLE
Allow for systems to configure the rector server by implementing an i…

### PR DESCRIPTION
**Pull request by rassmate**
…nterface ReactorServerConfigurer

### CODE REVIEW

**Review by jepp3 — APPROVED** [review]

**Review by JonasHallFnx — APPROVED** [review]

File: server/src/main/java/se/fortnox/reactivewizard/server/ReactorServerConfigurer.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package se.fortnox.reactivewizard.server;
+
+import reactor.netty.http.server.HttpServer;
+
+public interface ReactorServerConfigurer {
```
Comment (JonasHallFnx) [inline server/src/main/java/se/fortnox/reactivewizard/server/ReactorServerConfigurer.java:5]: Beware that ordering might be important for some configurations, as channel configurations in netty has a strict ordering.

**Review by tsfcode — APPROVED** [review]

File: server/src/main/java/se/fortnox/reactivewizard/server/ReactorServerConfigurer.java [hunk 2]
```diff
@@ -2,6 +2,24 @@
...
+     * Make custom configurations to the httpServer
+     * @param httpServer the server to be configured
+     * @return the configured server.
+     */
     HttpServer configure(HttpServer httpServer);
+
+    /**
+     * Returns which prio this configurer has.
+     *
+     * prio 1 will be executed before prio 2
```
Comment (JonasHallFnx) [inline server/src/main/java/se/fortnox/reactivewizard/server/ReactorServerConfigurer.java:19]: Which means that prio 2 will overwrite changes of prio 1 and therefore wins. No way to make things like this easy and intutive to understand... No action requested ;-)

**Review by JonasHallFnx — APPROVED** [review]

File: server/src/main/java/se/fortnox/reactivewizard/server/ReactorServerConfigurer.java (hunk 1 above)
Comment (rassmate) [reply server/src/main/java/se/fortnox/reactivewizard/server/ReactorServerConfigurer.java:8]: Added prio

File: server/src/main/java/se/fortnox/reactivewizard/server/ReactorServerConfigurer.java (hunk 2 above)
Comment (rassmate) [reply server/src/main/java/se/fortnox/reactivewizard/server/ReactorServerConfigurer.java:19]: 🧠

**Review by tsfcode — APPROVED** [review]

**Review by tsfcode — APPROVED** [review]

**Pull request merged**